### PR TITLE
Icon rows

### PR DESCRIPTION
**Pull request by jsmcgd**

### CODE REVIEW

File: web/app/themes/ppj/src/sass/landing-page.sass [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'colors'
 
 $color-main: $color-1
-$color-tertiary: $color-2
+$color-tertiary: rgb(25,25,52) // change to $color-dark after merge
```
Comment (ollietreend) [inline web/app/themes/ppj/src/sass/landing-page.sass]: Don't forget to change this after merge.

```
$color-tertiary: rgb(25,25,52) // change to $color-dark after merge
```

**Comment by ollietreend** [comment]
Should `dumbbell.png`, `eye.png` and `speech-bubbles.png` be included in this PR? It looks like the icons are implemented as SVGs. Incidentally, the SVGs look to be missing from this PR even though they're referenced in the code.

**Comment by jsmcgd** [comment]
The second commit is where the pngs were converted to svgs. Unfortunately I think I had copied the images to dest/ instead of src/ so they weren't picked up by git. That should be fixed now.

**Comment by jsmcgd** [comment]
To use icon rows, go to the editor for a page. Create a new page block, select a type of icon rows. Click 'Add row'. You can now select one of three icons for a row, enter a row title and enter some content for the row.

**Pull request merged**